idioms/angularNestJsLambda/backend/models/entity.module.ts.example

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+
+{{Entity module imports files}}
+
+@Module({
+  imports: [{{Entity module imports}}],
```

Comment:
Put the idiom file name here.
E.g.: Entity module imports - ./entity.module.imports.idiom
All idiom files must use .idiom extension.